lib/nerves_hub_link/configurator.ex

```diff
@@ -78,4 +81,16 @@ defmodule NervesHubLink.Configurator do
     {version_string, 0} = System.cmd("fwup", ["--version"])
     String.trim(version_string)
   end
+
+  defp add_fwup_public_keys(config) do
+    fwup_public_keys = NervesHubLink.Certificate.fwup_public_keys()
+
+    if fwup_public_keys == [] do
+      Logger.error("No fwup public keys were configured for nerves_hub_link.")
+      Logger.error("This means that firmware signatures are not being checked.")
+      Logger.error("nerves_hub won't allow this in the future.")
```

Comment:
Nerves hub doesn't allow this now.

Reply:
Should i remove the entire error then? or just that line?

Reply:
I think that it should log that `nerves_hub_link` won't work. I'm tempted to say that it should raise, but that requires more thought and I don't want that to hold up this PR (from me at least).